include/aspect/material_model/visco_plastic.h

```diff
@@ -319,6 +319,11 @@ namespace aspect
         */
         double max_yield_strength;
 
+        /**
+         * temperature gradient added to temperature used in the flow law.
+         */
+        double temperature_adiabat_for_viscosity;
```

Comment:
On it's own (e.g., excluding comments), the name `temperature_adiabat_for_viscosity` implies (to me) that the value is the absolute adiabatic temperature at a given point, rather than the temperature gradient.

I think renaming the variable to something like `adiabatic_temperature_gradient_for_viscosity` (or something similar), would be a bit clearer and could be deciphered even without a comment.